crates/sncast/src/starknet_commands/declare.rs

```diff
@@ -45,7 +49,7 @@ pub async fn declare(
     let contract_name: String = contract_name.to_string();
     let contract_artifacts = artifacts
         .get(&contract_name)
-        .ok_or(anyhow!("Failed to find artifacts in starknet_artifacts.json file. Please ensure you have enabled sierra and casm code generation in Scarb.toml"))?;
+        .ok_or(anyhow!("Failed to find artifacts in starknet_artifacts.json file. Please make sure you have specified correct package using `--package` flag and that you have enabled sierra and casm code generation in Scarb.toml."))?;
```

Comment:
It would be nice to include the contract name in the error message.

Smth like `Failed to find artifacts in starknet_artifacts.json file for {contract_name} contract`

Reply:
the diff showed, because I merged a few PRs and did not rebase - I have now done this, and contract name is included in the message.